Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,58 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 support new features in MongoDB 4.4. These features include:
```

Comment:
Should this be 4.4.0?

Reply:
Changed!

Reply:
Why did this comment disappear from the Files tab?